app/views/pages/dashboard.html.erb

```diff
@@ -80,9 +80,9 @@
 
     <h4>Resources</h4>
     <ul class="pretty">
+      <li><%= link_to 'Learn Chef', 'https://learnchef.com', target: 'blank' %></li>
```

Comment:
We have helpers you should be using for these. Use `learn_chef_url` instead of `'https://learnchef.com'`.

See https://github.com/opscode/supermarket/blob/master/app/helpers/custom_url_helper.rb.